dora/src/vm/known.rs

```diff
@@ -109,6 +110,22 @@ pub struct KnownTraits {
     pub zero: TraitId,
 }
 
+#[derive(Debug)]
+pub struct KnownAnnotations {
+    pub abstract_: AnnotationId,
+    pub final_: AnnotationId,
+    pub internal: AnnotationId,
+    pub override_: AnnotationId,
+    pub open: AnnotationId,
+    pub pub_: AnnotationId,
+    pub static_: AnnotationId,
+
+    pub test: AnnotationId,
+
+    pub cannon: AnnotationId,
```

Comment:
Do we still need this one?

Reply:
I'm not sure? The annotation exists and is used 200 times.

Reply:
If we don't need it, I'm happy to remove things, but I think this should be a separate PR, not mixed with this one.

Reply:
Agreed, let's keep this. I thought that use_cannon above would only be a leftover...